src/etl/NFTHelpers.cpp

```diff
@@ -165,7 +175,7 @@ getNFTokenBurnData(ripple::TxMeta const& txMeta, ripple::STTx const& sttx)
                 node.peekAtField(ripple::sfPreviousFields).downcast<ripple::STObject>();
             if (previousFields.isFieldPresent(ripple::sfNFTokens))
                 prevNFTs = previousFields.getFieldArray(ripple::sfNFTokens);
-        } else if (!prevNFTs && node.getFName() == ripple::sfDeletedNode) {
+        } else if (node.getFName() == ripple::sfDeletedNode) {
```

Comment:
I assume this logic is checked via unit tests and the change is correct but can you please explain what's going on here in a few words? 🙇

Reply:
The prevNFTS defines 170 line, it is impossible to be true here. So I removed the check. It must be false.

Reply:
Nice catch! checking if prevNFTs is empty is unnecessary in this case since it will be always be empty if it reaches this line